docs/sources/developers/http_api/org.md

```diff
@@ -282,11 +282,12 @@ Content-Type: application/json
 
 ## Admin Organizations API
 
-The Admin Organizations HTTP API does not currently work with an API Token. API Tokens are currently
-only linked to an organization and an organization role. They cannot be given the permission of server
-admin, only users can be given that permission. So in order to use these API calls you will have to
-use Basic Auth and the Grafana user must have the Grafana Admin permission (The default admin user
-is called `admin` and has permission to use this API).
+{{% admonition type="caution" %}}
+The Admin Organizations HTTP API currently does not work with service account tokens. They cannot be granted [Grafana server administrator permissions]({{< relref "../roles-and-permissions/#grafana-server-administrators" >}}).
+In order to use these API endpoints you have to use Basic Auth and the Grafana user must have the Grafana server administrator permission.
+
+The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
+{{% /admonition %}}
```

Comment:
- Removes "currently" to make the documentation more timeless: https://developers.google.com/style/timeless-documentation
- Updates the shortcode syntax to match our preferred form which has more consistent semantics (I will open a PR to rewrite all the existing ones later): https://grafana.com/docs/writers-toolkit/review/lint-prose/rules/#grafanashortcodes
- Replaces the broken link that uses the `relref` shortcode with a working link that follows our linking guidance: https://grafana.com/docs/writers-toolkit/write/links/
- Modified some language for readability and tone

```suggestion
{{< admonition type="caution" >}}
You can't authenticate to the Admin Organizations HTTP API with service account tokens.
Service accounts are limited to an organization and an organization role.
They can't be granted [Grafana server administrator permissions](/docs/grafana/<GRAFANA_VERSION>/administration/roles-and-permissions/#grafana-server-administrators).

To use these API endpoints you have to use Basic authentication and the Grafana user must have the Grafana server administrator permission.

The `admin` user that Grafana is provisioned with by default has permissions to use these API endpoints.
{{< /admonition >}}
```